src/Predict/jobData.js

```diff
@@ -33,6 +33,17 @@ const jobCards = {
     requiredChannels: ['nuclei', 'cytoplasm'],
     modelResolution: 0.5,
   },
+  polaris: {
+    file: 'tiff_stack_examples/MERFISH_example.tiff',
+    name: 'Polaris',
+    model: 'Polaris performs fluorescent spot detection.',
+    inputs: 'An image containing fluorescent spots generated by smFISH or similar assays.',
+    resolution: 'Polaris expects data that is approximately 100X (0.1 μm/pixel)',
+    thumbnail: 'thumbnails/FISH_example.png',
+    scaleEnabled: true,
```

Comment:
Would changing this to `false` eliminate the scale drop down entirely? I don't like including it if it's not actually functional.